README.md

```diff
@@ -43,6 +47,19 @@ If you are upgrading from a previous version of blender2ogre, and having problem
 * [Animations](http://www.youtube.com/watch?feature=player_embedded&v=5oVM0Lmeb68)
 * [Meshmoon: Video and text instructions how to install and use blender2ogre addon](http://doc.meshmoon.com/index.html?page=from-blender-to-meshmoon-part-1)
 
+## Exporting meshes
+To export a blender model: `File Menu > Export > Ogre3D (.scene & .mesh)`. If the selection is greyed out, the select the object to export from the blender Node tree (Scene collections). 
+
+If you have `OGRETOOLS_XML_CONVERTER` set to a `OgreXMLConverter.exe` path, then the export dialogue will display options relevant for the Ogre (v1) mesh format.
+
+If you have `OGRETOOLS_XML_CONVERTER` set to a `OgreMeshTool.exe` path, then the export dialogue will display options relevant for the OgreNext (v2) mesh format. If you do want to export in the OgreNext (v2.*) format, make sure in the `Export dialogue > General Settings > Mesh Export Version` is set to V2. The following combinations of parameters are recommended by the `OgreMeshTool.exe` help screen:
+* Recommended params for modern DESKTOP (with normal mapping): **OgreMeshTool -e -t -ts 4 -O puqs sourcefile [destfile]**
+* Recommended params for GLES2 (with normal mapping): **OgreMeshTool -e -t -ts 4 -O qs sourcefile [destfile]**
+* Recommended params for modern DESKTOP (without normal mapping): **OgreMeshTool -e -O puqs sourcefile [destfile]**
+* Recommended params for GLES2 (w/out normal mapping): **OgreMeshTool -e -O qs sourcefile [destfile]**
+You can check the arguments passed to `OgreMeshTool.exe` in the Blender console. (`Window Menu > Toggle System Console`)
+As of 18/09/2021, Blender will export the material format in a Ogre (V1) format. This is not compatible with OgreNext (V2.*). You will need to manually convert them to a material.json file.
```

Comment:
no need to put a date here, we can update the text once this changes.

Also, I am not sure whether this is correct. OgreNext can sort of load classical materials:
https://ogrecave.github.io/ogre-next/api/2.2/hlms.html#HlmsChangesMaterialsAlive

Reply:
If you export the mesh and material straight from blender and view it in OgreNext: the mesh will show but the material will not. The material needs to be hand modified and I thought I read that the .material files are being depreciated, replaced with [name].material.json. So I was trying to suggest that at the present, if you are targeting OgreNext, you need to hand modify the material files and while your at it, convert them to the *.json format for future proofing.

If you try to load a blender2ogre material file in OgreNext, The Ogre.log file throws this on initialising:
`22:12:05: OGRE EXCEPTION(3:RenderingAPIException): Fixed Function pipeline is no longer allowed nor supported. The material Test must use shaders in HlmsLowLevel::calculateHashFor at D:/ogre/ogre-next/OgreMain/src/OgreHlmsLowLevel.cpp (line 157)`
You can hand modifiy the material file too:
`hlms Test pbs
{
roughness 0.5
diffuse 3.14 3.14 3.14
specular 0.1 0.1 0.1
}`
And OgreNext won't throw an exception, but you still need to hand modify it as blender2Ogre gives it to you. And if you are going to hand modify it, you might as well aim for the future proof [file].material.json format right?

Anyway, that was my reasoning. What about the following:
`Blender will export the material format in a Ogre (V1) format. This is not compatible with OgreNext (V2.*). You should manually convert them to a material.json file. See the [Ogre Wiki: HLMS Materials](https://wiki.ogre3d.org/HLMS+Materials) for more information.`